lib/src/components/multi_date_content.dart

```diff
@@ -149,13 +156,18 @@ class _PartDayDraggableEventState extends State<PartDayDraggableEvent> {
               _lastDragDateTime =
                   _DragInfos.resolveOffset(context, details.globalPosition);
               widget.onDragUpdate!(_lastDragDateTime!);
+              _isMoved = true;
             }
-          : null,
+          : (_) => _isMoved = true,
       onDragEnd: widget.onDragEnd != null
           ? (details) {
-              widget.onDragEnd!(_lastDragDateTime!);
+              widget.onDragEnd!(_lastDragDateTime);
               _lastDragDateTime = null;
+              _isMoved = false;
             }
+          : (_) => _isMoved = false,
```

Comment:
By moving the conditional into the lambdas, the code seems a bit clearer to read:

```dart
      onDragUpdate: (details) {
        if (widget.onDragUpdate != null) {
          _lastDragDateTime =
              _DragInfos.resolveOffset(context, details.globalPosition);
          widget.onDragUpdate!(_lastDragDateTime!);
        }
        _isMoved = true;
      },
      onDragEnd: (details) {
        if (widget.onDragEnd != null) {
          widget.onDragEnd!(_lastDragDateTime);
          _lastDragDateTime = null;
        }
        _isMoved = false;
      },
```